3_acm/install_acm.sh

```diff
@@ -51,6 +51,9 @@ for CONTEXT in ${CLUSTERS}; do
     cat config-management.yaml.tpl | envsubst > config-management-${CONTEXT}.yaml
     kubectl apply -f config-management-${CONTEXT}.yaml
     rm config-management-${CONTEXT}.yaml
+    cat root-repo.yaml.tpl | envsubst > root-repo-${CONTEXT}.yaml
```

Comment:
I would suggest to rename the file to `root-sync.yaml.tpl`

Reply:
SG, updated the file name.